doc/nrf/gs_ins_windows.rst

```diff
@@ -83,20 +83,24 @@ The |NCS| consists of the following repositories:
 Every |NCS| release consists of a combination of these repositories at different revisions.
 
 .. note::
-   The latest state of development is on the master branches of the repositories.
-   Note that this might not be a stable state.
+   The latest state of development is on the master branch of the fw-nrfconnect-nrf repository.
    To ensure a usable state, the fw-nrfconnect-nrf repository defines the compatible states of the other repositories.
    However, this state is not necessarily tested.
+   For a higher degree of quality assurance, check out a tagged release.
 
    Therefore, unless you are familiar with the development process, you should always work with a specific release of the |NCS|.
 
 To manage the combination of repositories and versions, the |NCS| uses :ref:`zephyr:west`.
-The main repository, fw-nrfconnect-nrf, contains a `west manifest file`_ that defines the versions of all repositories.
-This means that fw-nrfconnect-nrf acts as manifest repository (see :ref:`Repository structure <zephyr:west-struct>`), while the other repositories are project repositories.
-The version of the manifest file defines what versions of the project repositories are checked out.
-
-In this way, you can decide to work with a specific |NCS| release either by initializing west with the manifest file for the release or by checking out the manifest file for the release and updating your project repositories to the corresponding state.
-Alternatively, you can work with the latest state of development by using the manifest file from the master branch of the fw-nrfconnect-nrf repository.
+The main repository, fw-nrfconnect-nrf, contains a `west manifest file`_ that defines the versions of all other repositories.
+This means that fw-nrfconnect-nrf acts as the manifest repository (see :ref:`Repository structure <zephyr:west-struct>`), while the other repositories are project repositories.
+The revision (in Git terms) of the fw-nrfconnect-nrf repository determines the contents of the manifest file.
+This file in turn defines what versions of the project repositories are checked out.
+This means that the full revision set of all repositories can be uniquely identified by a particular Git revision of the fw-nrfconnect-nrf repository.
+
+In this way, you can decide to work with a specific |NCS| release either by initializing a new west installation at a particular tag or by checking out the corresponding
+tag for a release in an existing installation and then updating your project repositories to the corresponding state with ``west``.
```

Comment:
No `` for west - this is the name of the tool, not the command that you type.

Reply:
Also, we shouldn't have line breaks within a sentence. 
(One line per sentence makes it easier to review/see changes and use the suggestion feature for changes.)

Reply:
Thanks, fixed